docs/lists.rst

```diff
@@ -0,0 +1,502 @@
+*****
+Lists
+*****
+
+Order... matters. And python lists make it easy to work with ordered data. Lists
+are a builtin data structure for storing and accessing objects which belong in
+a specific sequence. We will now learn how to create and use lists, and we will
+do so in a linear and orderly fashion.
+
+
+Simple uses
+###########
+
+There are two ways to create a list. One way is to use the ``list()``
+*constructor*. But a simpler and more common way is to use *brackets*.
+
+.. code-block:: python
+
+    samples = list()
+    # or ...
+    samples = []
+
+
+When creating a list, you can also pre-populate it with values. For example,
+let's create a list with the first few prime numbers:
+
+.. code-block:: python
+
+    prime_numbers = [2, 3, 5, 7, 11, 13]
+
+
+If you feel that these numbers are not enough, you can always add values later
+by using the ``append()`` method which allows you to add new values *to the end*
+of the list. Let's append the next two prime numbers: 17 and 19.
+
+.. code-block:: python
+
+    prime_numbers.append(17)
+    prime_numbers.append(19)
+
+    print(prime_numbers)
+
+
+If you display the list, you will see it contains the new values.
+Notice how lists preserve the order of the data; this is different from sets.
```

Comment:
new concept is introduced, might be misleading, so I suggest to keep it simple without making a parallel with set